### PR TITLE
HCO: Add tests for the new release-1.11 branch

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

**Comment by dharmit** [comment]
I have used k8s v1.27 & 1.28 instead of 1.26 & 1.27 in previous PRs. PTAL if something needs to be changed.

/cc @nunnatsa 
/cc @tiraboschi

File: github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+presubmits:
+  kubevirt/hyperconverged-cluster-operator:
+  - name: pull-hyperconverged-cluster-operator-e2e-k8s-1.27-centos9
```
Comment (nunnatsa) [inline github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml:3]: Please remove the `-centos9` suffix. This is not needed and confusing.

File: github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-shared-images: "true"
+    cluster: kubevirt-prow-workloads
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+      - image: quay.io/kubevirtci/golang:v20230626-e1b7af2
```
Comment (nunnatsa) [inline github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml:22]: maybe we can use something newer?

File: github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (nunnatsa) [inline github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml:56]: same

File: github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - "export TARGET=k8s-1.27-centos9 && automation/test.sh"
```
Comment (nunnatsa) [inline github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-1.11.yaml:27]: there is no `k8s-1.27-centos9`. please change to `k8s-1.27`

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Thanks for the PR @dharmit 

Please find some inline comments.

**Pull request merged**